### PR TITLE
Add Hungarian language

### LINKED ISSUE #491: Lang_hu.txt
[lang.hu.txt](https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/files/3505571/lang.hu.txt)

### PR DESCRIPTION
**Pull request by aubergine10**
New Hungarian language added - thanks JozsefHUNGepiM

> Note: Waiting for some feedback from contributor before marking PR ready.

Fixes #491

### CODE REVIEW

**Comment by krzychu124** [comment]
Testing in-game...

**Comment by aubergine10** [comment]
Ready for review :)

File: TLM/TLM/Resources/lang_hu.txt [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+Keybind_toggle_TMPE_main_menu Toggle TM: PE Főmenü
+Keybind_toggle_traffic_lights_tool Használja a „Forgalmi lámpák átkapcsolása” eszközt
+Keybind_use_lane_arrow_tool Használja a 'Lane Arrow' eszközt
+Keybind_use_lane_connections_tool Használja a „Lane Connections” eszközt
+Keybind_use_priority_signs_tool Használja az „Elsőbbségi jelek” eszközt
+Keybind_use_junction_restrictions_tool Használja a 'Junction Restrictions' eszközt
+Keybind_use_speed_limits_tool Használja a „Sebességkorlátozások” eszközt
+Keybind_lane_connector_stay_in_lane Maradjon a sávban
+Keybind_lane_connector_delete Tiszta sávcsatlakozások
+Keybinds keybinds
```
Comment (krzychu124) [inline TLM/TLM/Resources/lang_hu.txt]: Capitalize first letter

File: TLM/TLM/Resources/lang_hu.txt [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+Drivers_want_to_change_lanes_(only_applied_if_Advanced_AI_is_enabled): A járművezetők szeretnék megváltoztatni a sávot (csak akkor alkalmazzák, ha az Advanced AI engedélyezve van)
+Maintenance Karbantartás
+Very_often Nagyon gyakran
+Often Gyakran
+Sometimes Néha
+Rarely Ritkán
+Very_rarely Nagyon ritkán
+Only_if_necessary Csak ha szükséges
+Nodes_and_segments Csomópontok és szegmensek
+Lanes sávok
```
Comment (krzychu124) [inline TLM/TLM/Resources/lang_hu.txt]: Capitalize first letter

**Comment by aubergine10** [comment]
Pushed a commit with the requested changes

**Comment by aubergine10** [comment]
Ah, missed some `\ N`'s

**Comment by aubergine10** [comment]
Om that commit should fix all the `\n` stuff.

**Comment by krzychu124** [comment]
testing...

File: TLM/TLM/Resources/lang_hu.txt [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+Reload_global_configuration Töltse újra a globális konfigurációt
+Reset_global_configuration A globális konfiguráció visszaállítása
+General Tábornok
+Gameplay Gameplay
+Overlays Lefedések
+Individual_driving_styles Egyéni vezetési stílusok
+Evacuation_busses_may_ignore_traffic_rules Az evakuáló buszok figyelmen kívül hagyhatják a közlekedési szabályokat
+Evacuation_busses_may_only_be_used_to_reach_a_shelter Az evakuáló buszok csak menhelyhez juthatnak
+Vehicle_behavior A jármű viselkedése
+Policies_&_Restrictions házirendje
```
Comment (krzychu124) [inline TLM/TLM/Resources/lang_hu.txt]: That word should be capitalized too

Comment (aubergine10) [reply TLM/TLM/Resources/lang_hu.txt]: Done :)

**Pull request merged**